Avo.js

```diff
@@ -757,407 +791,380 @@ _avo_invoke_meta = function _avo_invoke_meta(env, type, messages, origin) {
-          "Schema Name": properties.schemaName,
-          "Branch Id": properties.branchId,
-          "Branch Name": properties.branchName,
-          "Client": sysClient,
-          "Version": sysVersion,
-          }, "yj_554q43i", "b457edd4df6bc469fbec7971b26fd46ce11aa405aa957ab11eb670edb91ca1b0")
-        : Promise.resolve(),
-        // destination CustomNodeJS
-        CustomNodeJS.logEvent(properties.userId_, "Cli Conflict Resolve Succeeded", {
-          "Cli Invoked by Ci": properties.cliInvokedByCi,
-          "Schema Id": properties.schemaId,
-          "Schema Name": properties.schemaName,
-          "Branch Id": properties.branchId,
-          "Branch Name": properties.branchName,
-          "Client": sysClient,
-          "Version": sysVersion,
-          }),
-      ]);
-    } else {
-      // do nothing
-      return new Promise((resolve) => { resolve(null); });
-    }
+        }, "yj_554q43i", "15ed68a02ce2178bd8be746d338dcec50f4e325069f03df6e5c164376fcb9c1f")
+      : Promise.resolve(),
+      // destination CustomNodeJS
+      CustomNodeJS.logEvent(properties.userId_, "Cli Conflict Resolve Succeeded", {
+        "Cli Invoked by Ci": properties.cliInvokedByCi,
+        "Schema Id": properties.schemaId,
+        "Schema Name": properties.schemaName,
+        "Branch Id": properties.branchId,
+        "Branch Name": properties.branchName,
+        "Client": sysClient,
+        "Version": sysVersion,
+        }),
+    ]);
+  } else {
+    // do nothing
+    return new Promise((resolve) => { resolve(null); });
   }
+}
 
-  exports.SignInError = SignInError;
-  exports.Client = Client;
-  exports.AuthenticationMethod = AuthenticationMethod;
-  exports.CliAction = CliAction;
-  exports.setAvoLogger = setAvoLogger;
-  exports.signedIn = signedIn;
-  exports.signInFailed = signInFailed;
-  exports.cliInvoked = cliInvoked;
-  exports.cliInstalled = cliInstalled;
-  exports.cliConflictResolveAttempted = cliConflictResolveAttempted;
-  exports.cliConflictResolveFailed = cliConflictResolveFailed;
-  exports.cliConflictResolveSucceeded = cliConflictResolveSucceeded;
-  exports.setSystemProperties_ = setSystemProperties_;
-  exports.setSystemProperties = setSystemProperties;
-  exports.setup_ = setup_;
-  exports.initAvo = initAvo;
-  exports.avoInspectorApiKey = avoInspectorApiKey;
-}(typeof exports === 'undefined' ? this.Avo = {} : exports));
+export default {
+  setAvoLogger: setAvoLogger,
+  SignInError: SignInError,
+  Client: Client,
+  AuthenticationMethod: AuthenticationMethod,
+  CliAction: CliAction,
+  signedIn: signedIn,
+  signInFailed: signInFailed,
+  cliInvoked: cliInvoked,
+  cliInstalled: cliInstalled,
+  cliConflictResolveAttempted: cliConflictResolveAttempted,
+  cliConflictResolveFailed: cliConflictResolveFailed,
+  cliConflictResolveSucceeded: cliConflictResolveSucceeded,
+  setSystemProperties: setSystemProperties,
+  initAvo: initAvo,
+  avoInspectorApiKey: avoInspectorApiKey,
+}
```

Comment:
Do we need to `default` export this map when we've already exported each function individually?

Reply:
Good question, I'm not sure! Something we can look into in the JS codegen for Avo Functions (this is an Avo generated file)

Reply:
Ohh of course!